app/src/features/admin/users/AccessRequestList.tsx

```diff
@@ -106,6 +131,28 @@ const AccessRequestList: React.FC<IAccessRequestListProps> = (props) => {
 
     setActiveReviewDialog({ open: false, request: null });
 
+    try {
+      await biohubApi.admin.sendGCNotification(
+        {
+          emailAddress: updatedRequest.data.email,
+          userId: updatedRequest.id
+        } as IgcNotifyRecipient,
```

Comment:
Can we save a couple lines with `const recipient: IgcNotifyRecipient = { emailAddress: ..., userId: ... }`?

Reply:
And/or potentially `const defaultBody = { body1: 'This is an automated message from the BioHub Species Inventory Management System', body2: ' ', footer: ' ' }`?